docs/admin/out-of-resource.md

```diff
@@ -35,20 +39,68 @@ summary API.
 | `imagefs.available` | `imagefs.available` := `node.stats.runtime.imagefs.available` |
 | `imagefs.inodesFree` | `imagefs.inodesFree` := `node.stats.runtime.imagefs.inodesFree` |
 
-Each of the above signals supports either a literal or percentage based value.  The percentage based value
-is calculated relative to the total capacity associated with each signal.
+The value for `memory.available` is derived from the cgroupfs instead of tools
+like `free -m`.  This is important because `free -m` does not work in a
```

Comment:
`free` uses `/proc/meminfo` which is not namespaced. Ideally cgroup namespaces would provide a container specific /proc/meminfo.